### ISSUE TITLE
Links broken on azul.rs

### ISSUE THREAD

**Issue by TomByrne** [fschutt/azul#269]
All of the links at `https://azul.rs/release/nightly` are broken.

**Comment by iMplode-nZ** [comment]
Additionallly, the readme link to the guide and api is ending with a `/`, which causes it to not work.
eg: `https://azul.rs/guide` vs `https://azul.rs/guide/`

**Comment by fschutt** [comment]
The release links are broken because there is no release yet. So far I've only uploaded the_ linux `libazul.so` and other files, for example the API header: https://azul.rs/release/nightly/files/azul.h

The links in the README have been fixed.

**Comment by NSoiffer** [comment]
All the file download links on https://azul.rs/release/1.0.0-alpha1 are broken. Since the documentation in many places says "use the pre-built libraries", it makes it rather hard to even try Azul...

**Comment by 2gn** [comment]
since the links are broken, libazul.so can only be retrieved by using the following link as mentioned.
https://azul.rs/release/nightly/files/libazul.so

**Comment by fschutt** [comment]
yeah, also that's a relatively old version by now. I should do another release.

**Comment by NSoiffer** [comment]
Sorry, I gave up and went with a different solution. Nonetheless, doing a new release and fixing the links is a good idea. If so, I'll consider using azul for a future project.
